python/src/LARS_doc.i.in

```diff
@@ -24,5 +24,36 @@ The sparse least squares approaches may be employed instead. Eventually a sparse
 PC representation is obtained, that is an approximation which only contains a
 small number of active basis functions.
 
-This class is not usable as is because it has sense only whithin the
-:class:`~openturns.FunctionalChaosAlgorithm`."
+Examples
+--------
+>>> import openturns as ot
+>>> from openturns.usecases import ishigami_function
+>>> im = ishigami_function.IshigamiModel()
+>>> # Create the orthogonal basis
+>>> polynomialCollection = ot.PolynomialFamilyCollection(im.dim)
+>>> for i in range(im.dim):
+...    polynomialCollection[i] = ot.OrthogonalUniVariatePolynomialFamily(
+...        ot.LegendreFactory()
+...    )
+>>> enumerateFunction = ot.LinearEnumerateFunction(im.dim)
+>>> productBasis = ot.OrthogonalBasis(
```

Comment:
No need to cast `ot.OrthogonalProductPolynomialFactory` into `ot.OrthogonalBasis`.
If you use the linear enumeration you don't even have to pass it to the constructor of `ot.OrthogonalProductPolynomialFactory` (but you may want to make it explicit)